ts/editor/NoteEditor.svelte

```diff
@@ -411,6 +417,15 @@ License: GNU AGPL, version 3 or later; http://www.gnu.org/licenses/agpl.html
             }
         }
 
+        // add event on window resize if event exist then do not add again
+        if (!window.onresize) {
+            window.onresize = () => {
+                if (imageOcclusionMode?.kind === "add") {
+                    resetIOImage(imageOcclusionMode.imagePath);
```

Comment:
I think this might be done more cleanly with Svelte's onMount(), perhaps in MaskEditor.svelte?